specification/k3po.control/SPEC.md

```diff
@@ -50,7 +50,7 @@ are formatted as follows:
 
 If the "content-length" header is specified, then a message body of the specified length must be added.
 
-Headers that are not understood / processed by a command must be ignored.
+Headers that are not understood / processed must result in an ERROR event if received by the server, or a closed connection if received on the client
```

Comment:
We probably need to relax this a little bit, so that writers of the protocol are precise and readers of the protocol are tolerant.